bin/juttle-engine

```diff
@@ -7,7 +7,7 @@ var engine = require('..');
 
 var defaults = {
     'port': 8080,
-    'root': '/',
+    'root': process.cwd(),
```

Comment:
Because thats where the `examples` dir is and we want to be able to run the example juttles as

```
http://localhost:../?path=/examples/index.juttle
```